code/WorkInProgress/rewardsLocker.dm

```diff
@@ -1179,3 +1179,63 @@
 					src.player.claimed_rewards.Add(S.type)
 			else
 				boutput(usr, "<span class='alert'>Redemption of \"[S.title]\" failed.</span>")
+////////navygreyhop////////
+/datum/achievementReward/ntso_commander
+	title = "(Skin set) Bland Bureaucrat"
+	desc = "Will change the skin of suits, naval jackets, and Head of personal uniforms."
+	required_medal = "Unperson"
+	once_per_round = 0
+
+	rewardActivate(var/mob/activator)
+		if (ishuman(activator))
+			var/mob/living/carbon/human/H = activator
+			var/succ = 0
```

Comment:
To make it clearer you can probably just use `var/success` here.

Also, should probably set it to `FALSE` rarher than 0

Reply:
i didnt want to mess this up and so just copy pasted the bluecap code